src/main/resources/rs117/hd/scene/tile_overrides.json

```diff
@@ -7285,6 +7289,28 @@
     "underlayIds": [ 56, 57 ],
     "groundMaterial": "MARBLE_1_GLOSS"
   },
+  {
+    "name": "Enchanted Valley Ledge",
+    "area": "ENCHANTED_VALLEY_LEDGES",
+    "overlayIds": [ 0 ],
+    "underlayIds": [ 0 ],
+    "groundMaterial": "EARTHEN_CAVE_FLOOR",
+    "uvScale": 0.9,
+    "uvOrientation": 256,
+    "minLightness": 30,
+    "maxSaturation": 111,
```

Comment:
`maxSaturation` caps at 7, so a value of 111 effectively does nothing, unless you also include a shift (and it normally shouldn't shift to values above 7 anyway, since that ends up bleeding into other parts of the packed color. I should've maybe included some safeguards here, since this can do some wonky stuff :sweat_smile: